schema.sql

```diff
@@ -0,0 +1,51 @@
+CREATE TABLE owners(
```

Comment:
- [ ] Greetings! 📜📩 Please ensure to incorporate the command for adding an "email" column to the "owners" table in the schema.sql file. This is essential for enhancing your database schema with an email column. 🛠️📧

```sh
-- Add an email column to your owners table
ALTER TABLE owners ADD COLUMN email VARCHAR(120);
```

Reply:
I'll add this as well. Appreciate that 👍🏻